src/components/Title.vue

```diff
@@ -1,10 +1,22 @@
 <template>
   <div>
-    <p class=" text-white rounded-sm px-2 mb-3 inline-block" :class="methodColor(method)">{{ method }}</p>
-    <p class="ml-3 inline-block font-mono">{{ url }}</p>
-    <p class="ml-3 inline-block text-xs" :class="statusColor(status)" data-cy="status">{{ status }}</p>
+    <div class="flex rounded-sm outline outline-slate-400 outline-1 p-2 mb-2">
+      <p class="font-mono font-bold">Method: </p>
+      <p class="px-2 ml-3 rounded-md inline-block text-white font-mono" :class="methodColor(method)">{{ method }}</p>
+    </div>
+
+    <div class="flex rounded-sm outline outline-slate-400 outline-1 p-2 mb-2">
+      <p class="font-mono font-bold">URL: </p>
+      <p class="ml-3 inline-block font-mono">{{ url }}</p>
+    </div>
+
+    <div class="flex rounded-sm outline outline-slate-400 outline-1 p-2 mb-2">
```

Comment:
I’d suggest showing this conditionally, so we see the stats box only when we have status:
```html
<div class="flex rounded-sm outline outline-slate-400 outline-1 p-2 mb-2" v-if="status">
```